dashboard/app/controllers/hint_view_requests_controller.rb

```diff
@@ -0,0 +1,14 @@
+class HintViewRequestsController < ApplicationController
+  def create
+    @hint_view_request = HintViewRequest.create(
+      script_id: params[:script_id],
+      level_id: params[:level_id],
+      user_id: current_user.id,
```

Comment:
hah, probably. Is `before_filter :authenticate_user!` the correct way to avoid that?